docs/en/setup/service-agent/java-agent/Application-toolkit-trace.md

```diff
@@ -18,6 +18,19 @@ _Sample codes only_
 
 * Add `@Trace` to any method you want to trace. After that, you can see the span in the Stack.
 * Add custom tag in  the context of traced method .
+
+* `ActiveSpan.error()` support user to make the span of traced method error
+* `ActiveSpan.error(String errorMsg)` support user to log some error message
+* `ActiveSpan.error(Throwable throwable)` support user to error a Throwable
+* `ActiveSpan.debug(String debugMsg)` support user to log some debug message
+* `ActiveSpan.info(String infoMsg)` support user to log some info message
```

Comment:
Add an info level log message in the current span.

Reply:
doc has updated